run.sh

```diff
@@ -29,11 +29,22 @@ if [ -n "$DEPLOY" ]; then
 else
   # its a build!
   export ACTION="build"
-  export ACTION_URL=$WERCKER_BUILD_URL
+  export ACTION_URL=$WERCKER_RUN_URL="https://app.wercker.com/APIPCS/$WERCKER_APPLICATION_NAME/runs/build/$WERCKER_BUILD_ID"
```

Comment:
I don't understand the intention here; is this supposed to set a default value?

Reply:
bug, fixed